test/components/core_spec.js

```diff
@@ -119,6 +124,23 @@ describe('Core', function() {
           expect(spy).to.have.been.calledWith('fullscreen nocursor')
         })
       })
+
+    })
+    describe('Multiple instances', () => {
+      it('shouldn\'t toggle one instance fullscreen state when another one stops', () => {
+        const newInstance = new Core({})
+        const fakeContainer1 = document.createElement('div')
+        fakeContainer1.setAttribute('id', 'fakeContainer1')
+        newInstance.el = fakeContainer1
+
+        expect(this.core.isFullscreen()).to.equal(false)
+        expect(newInstance.isFullscreen()).to.equal(false)
+
+        sinon.stub(Fullscreen, 'getFullscreenElement').returns(fakeContainer1)
+        expect(this.core.isFullscreen()).to.equal(false)
+        expect(newInstance.isFullscreen()).to.equal(true)
+
```

Comment:
There are a number of those empty lines at the end of blocks, please remove those.